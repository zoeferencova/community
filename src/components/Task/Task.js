import React, { Component } from "react";
import styles from "./Task.module.css";

export default class Task extends Component {
    render() {
        return (   
            <li className={styles.task}>{this.props.task}</li>        
        )
    }
}




