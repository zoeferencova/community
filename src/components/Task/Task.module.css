.task {
    padding: 7px 10px;
    font-size: 13px;
    border-radius: 5px;
    margin: 0px 8px 8px 0px;
}

.errands {
    background-color: #e4f2f2;
    color: #164d4d;
}

.supplies {
    background-color: #e4f0e4;
    color: #004d00;
}

.call {
    background-color: #f2f0fa;
    color: #130840;
}

.chat {
    background-color: #f5e6e6;
    color: #5e0202;
}

.dog {
    background-color: #faf9e6;
    color: #404040;
}

.other {
    background-color: #fcf5e8;
    color: #404040;
}