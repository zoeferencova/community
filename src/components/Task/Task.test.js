import React from "react";
import { shallow } from "enzyme";
import Task from "./Task";

describe("Task", () => {
    it("renders Task component", () => {
        const mockProps = {
            task: "Picking up supplies"
        }

        const wrapper = shallow(<Task {...mockProps} />);

        expect(wrapper).toMatchSnapshot();
    });
});