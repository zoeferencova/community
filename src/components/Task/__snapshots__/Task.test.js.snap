// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task renders Task component 1`] = `
<li
  className="task supplies"
>
  Picking up supplies
</li>
`;
